pipelines/live-1/main/tzcronjobber.yaml

```diff
@@ -0,0 +1,73 @@
+resources:
+- name: tools-image
+  type: docker-image
+  source:
+    repository: 754256621582.dkr.ecr.eu-west-2.amazonaws.com/cloud-platform/tools
+    tag: latest
+    aws_access_key_id: ((aws-live-1.access-key-id))
+    aws_secret_access_key: ((aws-live-1.secret-access-key))
+- name: cronjobber-image
+  type: docker-image
+  source:
+    repository: 754256621582.dkr.ecr.eu-west-2.amazonaws.com/cloud-platform/tools
```

Comment:
Where is the repository created? Shouldn't it be defined under `tzcronjobber/` ?

Reply:
I now realise that I originally misread the repo URL. I did not see that the image was pushed as `tools:cronjobber` but thought it's pushed as `cronjobber:latest`. Thanks for changing this to its own repository!

What I meant, however, is that the actual ECR repository resource is not defined anywhere. In this repo, we can define resources with terraform (like in environments) by creating a directory using the same name as the pipeline's definition, ie.: `tzcronjobber/` - we have the `build-environments` pipeline use that to create the `kubeconfig` file it uses (see the bootstrap pipeline for how it works).

Reply:
done, one more read pls